safehttp/form.go

```diff
@@ -212,9 +213,19 @@ func (f *MultipartForm) File(param string) []*multipart.FileHeader {
 	if !ok {
 		return nil
 	}
+	sanitise(fh)
```

Comment:
Shouldn't this be done once when the form is parsed and returned by IncomingRequest?

Reply:
moved sanitization to constructor